src/components/item/iterator.js

```diff
@@ -324,6 +369,42 @@ class ItemIterator extends Iterator {
   }
 }
 
+const DropTargetSpec = {
+  drop({ list, onItemImport }, monitor) {
```

Comment:
Careful! Both `drop` and `canDrop` have changed since you copied them -- please use the latest version here or we're losing support for dropping URLs.

Reply:
Here I copied what was on project view: returning true as default.

Reply:
done
I brought the changes from ProjectView done on https://github.com/tropy/tropy/commit/019f4e2c3bd07efdac4b1f738780dcefa40f4c04 to the `drop` & `canDrop` methods

Reply:
There were more changes than just this one commit. You need to compare the branches: what this PR removes from project view needs to be identical to what is now in the item iterator

Reply:
You're  removing this  code:
![Screenshot from 2019-11-18 17-26-29](https://user-images.githubusercontent.com/325102/69070607-b9c6bb00-0a28-11ea-94e5-43d2286dcb09.png)

You're  adding  this  code:
![Screenshot from 2019-11-18 17-27-21](https://user-images.githubusercontent.com/325102/69070625-c1865f80-0a28-11ea-834a-d6bc7a82d383.png)
![Screenshot from 2019-11-18 17-26-59](https://user-images.githubusercontent.com/325102/69070637-c64b1380-0a28-11ea-8073-7ebaaebdb191.png)